scripts/test

```diff
@@ -132,6 +132,21 @@ elif [ "$ARCH" = "arm32" -o "$ARCH" = "arm64" ]; then
 
 	file apps/openssl/.libs/openssl
 
+elif [ "$ARCH" = "loong64" ]; then
+	sudo apt install -y qemu-user-static binfmt-support g++-loongarch64-linux-gnu
+	sudo ln -sf /usr/loongarch64-linux-gnu/lib/ld-linux-loongarch-lp64d.so.1 /lib
+
+	CC=loongarch64-linux-gnu-gcc
```

Comment:
These variable assignments are not really doing anything; should be able to remove. --host on configure will figure this all out automatically. Look at how mips and arm doit.

Reply:
I was unable to get it to build locally without these, it always built for x86-64 instead. It may be an issue just on my machine though, I can try removing to see if it works properly in GitHub Actions.

Reply:
If dropping it works I'm fine with that, but I have definitely seen cross-compilation fail due to using the host's default compiler (it's been a while - macos arm64 seems to have hit this from time to time).